src/zen/websocket_server.cpp

```diff
@@ -488,6 +507,66 @@ class WsHandler
         return OK;
     }
 
+    int sendTopQualityCertificatesFromScid(const std::string& scIdString, const std::string& clientRequestId)
+    {
+        uint256 scId;
+        scId.SetHex(scIdString);
+        
+        CCoinsViewCache &view = *pcoinsTip;
+        
+        UniValue mempoolTopQualityCert(UniValue::VOBJ);
+        UniValue chainTopQualityCert(UniValue::VOBJ);
+
+        {
+            LOCK(cs_main);
+            if (!view.HaveSidechain(scId)) {
+                LogPrint("ws", "%s():%d - sidechain id not found[%s]\n", __func__, __LINE__, scIdString);
+                return INVALID_PARAMETER;
+            }
+        
+            if (mempool.hasSidechainCertificate(scId))
+            {
+                const uint256& topQualCertHash = mempool.mapSidechains.at(scId).GetTopQualityCert()->second;
+                const CScCertificate& topQualCert = mempool.mapCertificate.at(topQualCertHash).GetCertificate();
+                CDataStream ssBlock(SER_NETWORK, PROTOCOL_VERSION);
+                ssBlock << topQualCert;
+                std::string certHex = HexStr(ssBlock.begin(), ssBlock.end());
+     
+                mempoolTopQualityCert.push_back(Pair("quality", topQualCert.quality));
+                mempoolTopQualityCert.push_back(Pair("certHash", topQualCertHash.GetHex()));
+                mempoolTopQualityCert.push_back(Pair("rawCertificateHex", certHex));
+                mempoolTopQualityCert.push_back(Pair("fee", topQualCert.mainchainBackwardTransferRequestScFee));
```

Comment:
Mempool Cert fee is different. Fees are the coins amount diff between inputs and outputs.
There is an info about cert fees in the mempool.